pkg/client/cli/cliutil/daemon.go

```diff
@@ -53,53 +46,7 @@ func launchDaemon(ctx context.Context, dnsIP string) error {
 	if err != nil {
 		return err
 	}
-
-	args := []string{client.GetExe(), "daemon-foreground", logDir, configDir, dnsIP}
-	if os.Geteuid() != 0 {
-		// If we're going to be prompting for the `sudo` password, we want to first provide
-		// the user with some info about exactly what we're prompting for.  We don't want to
-		// use `sudo`'s `--prompt` flag for this because (1) we don't want it to be
-		// re-displayed if they typo their password, and (2) it might be ignored anyway
-		// depending on `passprompt_override` in `/etc/sudoers`.  So we'll do a pre-flight
-		// `sudo --non-interactive true` to decide whether to display it.
-		//
-		// Note: Using `sudo --non-interactive --validate` does not work well in situations
-		// where the user has configured `myuser ALL=(ALL:ALL) NOPASSWD: ALL` in the sudoers
-		// file. Hence the use of `sudo --non-interactive true`. A plausible cause can be
-		// found in the first comment here:
-		// https://unix.stackexchange.com/questions/50584/why-sudo-timestamp-is-not-updated-when-nopasswd-is-set
```

Comment:
This comment went away... for no real reason?  IMO it was a very helpful comment.